src/pages/tasks/TaskDescriptionPage.js

```diff
@@ -48,6 +48,17 @@ function TaskDescriptionPage(props) {
 
     const inputRef = useRef(null);
 
+    // Same as NewtaskDescriptionPage, use the selection to place the cursor correctly if there is prior text
```

Comment:
Who knows what this page might use in the future or if it will continue to exist at all. Referencing it here does not add anything.